### PR TITLE
Fix ATOTM rsk event handler, fix RSK uptime calculation and rename RSK damage event 

### PR DESCRIPTION
**Pull request by trevorm4**
- Renamed the RSK damage event from SPELLS.RISING_KICK_SECOND to SPELLS.RISING_KICK_DAMAGE (old name meant zero sense to any human looking at it)
- Removed rsk event fabrication RSK module as its handled by SpellUsable now (also fixed in SL branch #5200)
- Fixed ATOTM module as the event listener was listening to talent ID on a damage event listener (instead of the RSK damage id)

### CODE REVIEW

File: src/analysis/retail/monk/mistweaver/modules/spells/RisingSunKick.tsx [hunk 1]
```diff
@@ -41,9 +41,6 @@ class RisingSunKick extends Analyzer {
 
     if (this.lastBOK > this.lastRSK) {
       this.rskResets += 1;
-
-      this.spellUsable.endCooldown(TALENTS_MONK.RISING_SUN_KICK_TALENT.id);
-      this.spellUsable.beginCooldown(event, TALENTS_MONK.RISING_SUN_KICK_TALENT.id);
```
Comment (kfinch) [inline src/analysis/retail/monk/mistweaver/modules/spells/RisingSunKick.tsx:46]: I thought you added these lines recently - is this a merge mistake?

Comment (trevorm4) [reply src/analysis/retail/monk/mistweaver/modules/spells/RisingSunKick.tsx:46]: The first PR was on the shadowlands branch, but this is for DF branch. Is there some syncing across them that I should've been using?

Comment (kfinch) [reply src/analysis/retail/monk/mistweaver/modules/spells/RisingSunKick.tsx:46]: No - I might be getting mixed up I thought these lines were supposed to be added not removed

Comment (trevorm4) [reply src/analysis/retail/monk/mistweaver/modules/spells/RisingSunKick.tsx:46]: Nope, they are what was causing the issue. It was restarting the CD here, but also getting restarted in SpellUsable.tsx which is what causes the issue.

Comment (kfinch) [reply src/analysis/retail/monk/mistweaver/modules/spells/RisingSunKick.tsx:46]: Apologies then, in you go

**Pull request merged**